specs/dwritecore/DWriteCore.md

```diff
@@ -0,0 +1,255 @@
+# DWriteCore
+
+DWriteCore is the Project Reunion version of DirectWrite, the text rendering and layout
+component that ships in Windows. DWriteCore implements the same API as DirectWrite,
+with a few additions described in this document. See the [DirectWrite documentation](https://docs.microsoft.com/en-us/windows/win32/api/_directwrite/)
+for more information.
+
+For an application that already uses DirectWrite, switching to DWriteCore requires minimal changes:
+
+ - Add a reference to the Project Reunion package.
+ - Include dwrite_core.h instead of dwrite_3.h.
+ - Link to DWriteCore.lib instead of DWrite.lib.
+
+In return, the application gets the benefits of Project Reunion, namely, access to the newest APIs
+and functionality regardless of what version of Windows your customer is running.
```

Comment:
DISCUSSION: There's open issues with how DWriteCore will target other renders, including additional future APIs to target non-GDI things.

Reply:
DISCUSSION: Much of the implementation is about using per-process isolated content, like not accessing system caches or the global list of caches.  A core use is letting an app allow-list fonts. A browser might use that to prevent fingerprinting, or enforcing access control from a very low rights render process vs a display or management process.

Reply:
I updated the comments to hopefully add more clarity.